### PR TITLE
Add warnings to LSP7 & LSP8 compatible contracts

### PR DESCRIPTION
**Pull request by b00ste**
# What does this PR introduce?

This PR introduces warnings about deprecating the LSP7 & LSP8 compatible contracts.

### CODE REVIEW

File: docs/contracts/overview/DigitalAssets.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ The **Digital Asset (Token and NFT 2.0)** contracts are the newest advanced vers
 
 ## Comparisons with ERC20 / ERC721
 
+:::danger beware
+
+The LSP7 contracts Compatible to ERC20 and LSP8 contracts Compatible with ERC721 are being deprecated and will be deleted from [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts) repository. Howether if you want to stil use/maintain them, they will remain available in the version [`0.14.0`](https://github.com/lukso-network/lsp-smart-contracts/releases/tag/lsp-smart-contracts-v0.14.0).
```
Comment (Hugoo) [inline docs/contracts/overview/DigitalAssets.md:17]: compatible _to_ or compatible _with_ ? You are using both here.

Why uppercase on the word Compatible?

File: docs/contracts/contracts/LSP8IdentifiableDigitalAsset/presets/LSP8CompatibleERC721Mintable.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 
 # LSP8CompatibleERC721Mintable
 
+:::danger beware
+
+This contract is being deprecated and will be deleted from [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts) repository. Howether if you want to stil use/maintain it, you will sill be able to find it in the version [`0.14.0`](https://github.com/lukso-network/lsp-smart-contracts/releases/tag/lsp-smart-contracts-v0.14.0).
```
Comment (Hugoo) [inline docs/contracts/contracts/LSP8IdentifiableDigitalAsset/presets/LSP8CompatibleERC721Mintable.md:8]:
```suggestion
This contract is being deprecated and will be deleted from [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts) repository. However, if you want to still use/maintain it, you will sill be able to find it in the version [`0.14.0`](https://github.com/lukso-network/lsp-smart-contracts/releases/tag/lsp-smart-contracts-v0.14.0).
```

File: docs/contracts/contracts/LSP8IdentifiableDigitalAsset/extensions/LSP8CompatibleERC721.md [hunk 3]
```diff
@@ -3,6 +3,12 @@
 
 # LSP8CompatibleERC721
 
+:::danger beware
+
+This contract is being deprecated and will be deleted from [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts) repository. Howether if you want to stil use/maintain it, you will sill be able to find it in the version [`0.14.0`](https://github.com/lukso-network/lsp-smart-contracts/releases/tag/lsp-smart-contracts-v0.14.0).
```
Comment (Hugoo) [inline docs/contracts/contracts/LSP8IdentifiableDigitalAsset/extensions/LSP8CompatibleERC721.md:8]: typo check other comment

File: docs/contracts/contracts/LSP7DigitalAsset/presets/LSP7CompatibleERC20Mintable.md [hunk 4]
```diff
@@ -3,6 +3,12 @@
 
 # LSP7CompatibleERC20Mintable
 
+:::danger beware
+
+This contract is being deprecated and will be deleted from [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts) repository. Howether if you want to stil use/maintain it, you will sill be able to find it in the version [`0.14.0`](https://github.com/lukso-network/lsp-smart-contracts/releases/tag/lsp-smart-contracts-v0.14.0).
```
Comment (Hugoo) [inline docs/contracts/contracts/LSP7DigitalAsset/presets/LSP7CompatibleERC20Mintable.md:8]: typo stil and however

File: docs/contracts/contracts/LSP7DigitalAsset/extensions/LSP7CompatibleERC20.md [hunk 5]
```diff
@@ -3,6 +3,12 @@
 
 # LSP7CompatibleERC20
 
+:::danger beware
+
+This contract is being deprecated and will be deleted from [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts) repository. Howether if you want to stil use/maintain it, you will sill be able to find it in the version [`0.14.0`](https://github.com/lukso-network/lsp-smart-contracts/releases/tag/lsp-smart-contracts-v0.14.0).
```
Comment (Hugoo) [inline docs/contracts/contracts/LSP7DigitalAsset/extensions/LSP7CompatibleERC20.md:8]: typo

**Review by Hugoo — CHANGES_REQUESTED** [review]
be careful with typos

Did you also update the text on the lsp smart contract repo?

If yes, could you please add the PR link in the PR description, this would save time.

If not, please make PR on other repo + add link here

thks

**Review by YamenMerhi — APPROVED** [review]

**Review by Hugoo — APPROVED** [review]

**Pull request merged**